exercises/resistor-color-master/metadata.toml

```diff
@@ -0,0 +1,3 @@
+title = "Resistor Color Master"
+blurb = "Convert color codes, as used on resistors, to a human-readable label."
+source = "BethanyG, meatball133"
```

Comment:
There may be an excerpt somewhere from Maud and @ErikSchierboom

Reply:
I found this in `resistor-color-duo`:

>   "blurb": "Convert color codes, as used on resistors, to a numeric value.",
  "source": "Maud de Vries, Erik Schierboom",
  "source_url": "https://github.com/exercism/problem-specifications/issues/1464"

Which I do think we should add in.  Is that what you had in mind, @SleeplessByte?